fab/fabfile.py

```diff
@@ -436,6 +436,22 @@ def apply_patch(patchfile=None):
     silent_services_restart(use_current_release=True)
 
 
+@task
+def reverse_patch(patchfile=None):
+    """
+    Used to reverse a git patch created via `git format-patch`. Usage:
+
+        fab <env> reverse_patch:patchfile=/path/to/patch
+
+    Note: Only use this when absolutely necessary.
```

Comment:
Yeah NIC has caused so many headaches 😖 . For reference `-R` means reverse the specified patch, `--unsafe-paths` makes apply a better gnu patch (instead of trying to create a git commit), and `--directory` is just the root dir from which the patch should be applied `code_current`. will elaborate in the docstring